commit 4c7a36d3f5e48b05b5e08466ce22cf260a7a0575 (refs/changes/76/476/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2016-09-15T16:36:54-07:00 (3 years, 1 month ago)
    
    DOCKER-931 Docs: Remove obsolete limits on maximum TCP and UDP exposed ports

diff --git a/docs/api/features/networks.md b/docs/api/features/networks.md
index 0110508..5e939d0 100644
--- a/docs/api/features/networks.md
+++ b/docs/api/features/networks.md
@@ -39,10 +39,9 @@ container will receive an external IP address that is reachable over the public
 internet, and [Cloud Firewall](https://www.joyent.com/developers/firewall/)
 rules are automatically created that allow incoming traffic to the appropriate
 ports from any IP address. For `-P`, this means all ports that the VM's image
-exposes. For `-p`, this means all ports specified as arguments, up to a limit
-of 128 TCP ports and 128 UDP ports. Remapping ports with `-p` (eg, `-p 80:8080`)
-is not supported at this time. Follow [DOCKER-341](http://smartos.org/bugview/DOCKER-341)
-for updates.
+exposes. For `-p`, this means all ports specified as arguments. Port remapping
+(eg, `-p 80:8080`) is not supported at this time. Follow
+[DOCKER-341](http://smartos.org/bugview/DOCKER-341) for updates.
 
 If fabric networking is not enabled, all docker containers are provisioned with
 a nic on the 'external' network by default.
